Dockerfile

```diff
@@ -249,26 +249,28 @@ RUN set -x \
 	&& go build -v -o /usr/local/bin/rsrc github.com/akavel/rsrc \
 	&& rm -rf "$GOPATH"
 
+# Upload docker source
+COPY . /go/src/github.com/docker/docker
+
```

Comment:
Not a fan of having to put this before the last two commands. With it being here, it requires that the last two steps are always run. It needs to be here so that they directory is available when the next 2 steps are run. Open to other options.